spec/relevance/basic_search_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+RSpec.describe "Searches with format set to 'Score' "do
```

Comment:
@dkinzer This seems specific to tul_cob and musical scores.  Do we have any records with a format of score in the fixture files here?

Reply:
Need to update that verbage ... I'm only doing two very simple searches in this repo.  One for everything and one for a specific document.